### PR TITLE
ISPN-15923 More detailed metrics in CacheMgmtInterceptor

### PR DESCRIPTION
**Pull request by pruivo**
* Add counters for single key operations by ownership
* Add transaction prepare/commit/rollback operation times

https://issues.redhat.com/browse/ISPN-15923

### CODE REVIEW

File: core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java [hunk 1]
```diff
@@ -121,7 +121,7 @@ public Collection<MetricInfo> getCustomMetrics(GlobalMetricsConfiguration config
          attributes.add(MetricUtils.<RpcManagerImpl>createFunctionTimer("CrossSiteReplicationTimes",
                "Cross Site Replication Times",
                (rpcManager, timerTracker) -> rpcManager.xSiteMetricsCollector.registerTimer(timerTracker),
-               TimeUnit.NANOSECONDS, null));
+               TimeUnit.MICROSECONDS, null));
```
Comment (jabolina) [inline core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java:124]: Does this also change the output unit in the endpoint?

Comment (pruivo) [reply core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java:124]: I'm tempted to remove the parameter from the method and store everything in nanoseconds. WDYT?

Comment (jabolina) [reply core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java]: +1. My IDE shows every place is using nanoseconds already, and looking at the diff, it simplifies a lot of places.

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Thanks, @pruivo!

**Pull request merged**